resource_type.go

```diff
@@ -60,7 +60,7 @@ func (t ResourceType) validate(raw []byte, method string) (ResourceAttributes, *
 	var attributes map[string]interface{}
 	var scimErr *errors.ScimError
 	switch method {
-	case http.MethodGet, http.MethodPut:
+	case http.MethodPost, http.MethodPut:
```

Comment:
Is the `validate` method even called with methods other than `POST` and `PUT`?

Reply:
No, it is not.

Reply:
> No, it is not.

Maybe we can just leave the `switch` out in that case?